virtualization/windowscontainers/manage-containers/container-networking.md

```diff
@@ -229,6 +249,10 @@ To set a VLAN ID for a network, use the option, `-o com.docker.network.windowssh
 ```none
 C:\> docker network create -d transparent -o com.docker.network.windowsshim.vlanid=11 MyTransparentNetwork
 ```
+**Note:** Ensure that your host network adapter (physical) is in trunk mode to enable the flow of tagged traffic between container host endpoints and endpoints that are external to your host.
```

Comment:
...is in trunk mode to enable all tagged traffic to be processed by the vSwitch with the vNIC (container endpoint) port in access mode on the correct VLAN."